docs/architecture/email.md

```diff
@@ -44,3 +44,56 @@ Notes:
 2. EF-CMS uses Amazon SES Easy DKIM, [which uses a 1024-bit key](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/send-email-authentication-dkim-easy.html).
 3. Amazon SES does not support DNSSEC on its Easy DKIM signing domains.
 4. Encrypting emails with S/MIME would reduce the effectiveness of email notifications, as some recipients would not be able to decrypt the emails. No sensitive information is sent in email notifications.
+
+### Verifying SES Email
+
+At present this is a manual process as a human needs to click on the link in the email in order to verify ownership. We verify ownership so that we can use that verified email's ARN in [our Cognito configuration](../../web-api/terraform/template/cognito.tf). This lets us send more email than Cognito's very low quotas allow. 
+
+Here are the steps required to verify a new email:
+
+1. Need to create an s3 bucket (e.g. `mail-verification.example.com`)
+2. Need to grant SES permission to write to it. Replace `BUCKET-NAME` and `AWSACCCOUNTID` in the following policy, and apply it to the newly created bucket:
+
+    ```json
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Sid": "AllowSESPuts",
+                "Effect": "Allow",
+                "Principal": {
+                    "Service": "ses.amazonaws.com"
+                },
+                "Action": "s3:PutObject",
+                "Resource": "arn:aws:s3:::BUCKET-NAME/*",
+                "Condition": {
+                    "StringEquals": {
+                        "aws:Referer": "AWSACCOUNTID"
+                    }
+                }
+            }
+        ]
+    }
+    ```
+
+3. In SES, create a rule set: (e.g., `confirm_email_helper`)
+4. Need to create a Rule in that has the following attributes: (Replace `BUCKET-NAME`)
+
+   ```json
+   {
+      "Name": "Confirm Email Rule",
+      "Enabled": true,
+      "Actions": [
+        {
+          "S3Action": {
+            "BucketName": "BUCKET-NAME"
+          }
+        }
+      ]
+    }
+    ```
+
+5. Need to make the Rule Set active
+6. Use SES to send a verification email to that account.
+7. Check the S3 bucket. It should have an item in there with a link to confirm the Email Address. Follow the link, and the email address should be verified.
+8. When complete, delete the S3 Bucket, SES Rule, and SES Rule Set. Future deploys should rely on the verified email.
```

Comment:
I’m a bit confused here — will this store emails in S3 from this point forward? This looks to be nearly directly the example from *[How can I use Amazon SES to receive inbound emails, and then store those emails on Amazon S3?](https://aws.amazon.com/premiumsupport/knowledge-center/ses-receive-inbound-emails/)*. 

Is this the desired functionality? :scratching-head:

Reply:
Maybe we should grab a few minutes to pair. I’m definitely an expert on SES.

Reply:
Yes, this is the desired functionality. And good find, I did use that as a guide for setting this up.

This is what lets you verify the email address without actually requiring an inbox or to forward the user. 

And in fact, I think we're going to use this functionality by setting practitioner and petitioner emails to `md5(${originalEmail})` + `@mig.ef-cms.ustaxcourt.gov` during the next dry-run. This will help us:

- verify that no emails are sent during the migration.
- verify that the welcome emails are all sent
- verify that service emails are being sent properly.

... all by checking the S3 bucket.

Reply:
Gotcha! I thought "This is what lets you verify the email address without actually requiring an inbox or to forward the user." meant you were intending to use this S3 "inbox" as a way to verify an email address, rather than an S3 destination — I can’t believe this is a desired and intended process from AWS, but… /shruggie.

👍 🎉